met-web/src/components/survey/building/index.tsx

```diff
@@ -261,6 +278,34 @@ const SurveyFormBuilder = () => {
             <Grid item xs={12}>
                 <FormBuilder handleFormChange={handleFormChange} savedForm={formDefinition} />
             </Grid>
+            <PermissionsGate scopes={[SCOPES.createSurvey]} errorProps={{ disabled: true }}>
```

Comment:
I think the PermissionGate will pass the disabled prop to its immediate child in this case Grid. But that won't have an effect. Maybe bring the PermissionGate to wrap the Switch instead